builtin/mainmenu/tab_online.lua

```diff
@@ -56,18 +56,23 @@ local function get_sorted_servers()
 end
 
 -- Persists the selected server in the "address" and "remote_port" settings
+
+local function is_selected_fav(server)
+	local address = core.settings:get("address")
+	local port = tonumber(core.settings:get("remote_port"))
+
+	for _, fav in ipairs(serverlistmgr.get_favorites()) do
+			if address == fav.address and port == fav.port then
+				return true
+			end
+		end
```

Comment:
1. Indentation is wrong here (need to dedent the preceding 4 lines by 1)
2. I would add a `return false` for minor style reasons (conceptually this returns a bool: whether the selected server is a favorite; nil technically has the correct falsiness but it doesn't sit right with me considering our code style)